hypertrace-ingester/src/test/resources/configs/hypertrace-ingester/application.conf

```diff
@@ -0,0 +1,19 @@
+main.class = org.hypertrace.ingester.HypertraceIngester
+
+service.name = hypertrace-ingester
+service.admin.port = 8099
+
+sub.topology.names = ["span-normalizer", "raw-spans-grouper", "hypertrace-trace-enricher", "all-views"]
+
+precreate.topics = false
+precreate.topics = ${?PRE_CREATE_TOPICS}
```

Comment:
Is this duplicated of the previous line?

Reply:
Will remove this.